src/pages/guides/troubleshooting/faq.md

```diff
@@ -35,3 +51,25 @@ Content publishing: For the same reason above, you have the ability to pre-defin
 ### My user base consists of students/minors. How can I be assured they get appropriate content?
 
 The embedded Adobe Express full editor can be setup to launch student safe content even before a student is signed into the embedded experience. More information about [Adobe & Student Privacy](https://www.adobe.com/privacy/student-policy.html).
+
+### How many domains can be added with a single API key for a service?
```

Comment:
It will be better to add an example of the wildcard subdomains . E.g. If you want multiple subdomains foo.abc.com, bar.abc.com you can add *.abc.com to the allow list.

Reply:
Made the changes. Thank you.